src/views/News/index.jsx

```diff
@@ -260,6 +254,7 @@ const StudentNews = () => {
     }
 
     setOpenPostActivity(false);
+    handleWindowClose();
```

Comment:
If you call `handleWindowClose()`, that will call `setOpenPostActivity(false)` again, so I would remove the first call.
```suggestion
    handleWindowClose();
```
Additionally, `handleWindowClose()` will clear out all of the state values regarding the new entry, so it should only be called after the student news edit succeeded, not if it failed. I'd move the call inside of the `else` block above.